lib/request_handler/fieldsets_parser.rb

```diff
@@ -6,18 +6,19 @@ module RequestHandler
   class FieldsetsParser
     def initialize(params:, allowed: {}, required: [])
       @params = params
+      allowed = allowed.reject { |k, v| v.is_a?(FalseClass) }
```

Comment:
lets not check for the class here but simply compare to false. Also we can use the mutable version of reject.

```
allowed.reject! { |k, v| v == false }
```